docs/whats-new/changelog.md

```diff
@@ -1,7 +1 @@
 # Changelog: What's new with stravalib
-
-
-:::{include} ./changelog.md
```

Comment:
i think this include was causing the recursion issue. if that is the case i can work on that later.  it was an issue with myst parses but i don't understand what the issue is -- yet.